libs/test/SalesforceHybridTest/src/com/salesforce/androidsdk/phonegap/SmartStoreLoadJSTest.java

```diff
@@ -63,7 +64,9 @@
 
     @BeforeClass
     public static void runJSTestSuite() throws InterruptedException {
-        JSTestCase.runJSTestSuite(JS_SUITE, data(), 60);
+        // FIXME Update this when we stop using ARM Emulators
+        int timeout = Build.SUPPORTED_ABIS[0].contains("armeabi") ? 180 : 60;
+        JSTestCase.runJSTestSuite(JS_SUITE, data(), timeout);
```

Comment:
Why not just bump it up to `180` across the board?

Reply:
Didn't want to force a 3x timeout on multiple tests for the devices that definitely don't need it.  But since it seems unlikely these tests will break in that exact way I will simplify it to just 180.